### ISSUE TITLE
UX: ls -> list, rm --> remove

### ISSUE THREAD

**Issue by strk** [MichaelMure/git-bug#238]
For consistency, I think the `ls` command should be `list` and the `rm` command (for bridges) should be `remove`. This is consitent with `create`, `comment` etc..

**Comment by MichaelMure** [comment]
Hmm, I'm not sure what you mean. Unless I'm missing something, git-bug always use the short `ls` or `rm`. 

It's certainly possible to add alias for the full letter version though.

**Comment by A-Hilaly** [comment]
Yes cobra support aliases we can certainly support having the full letter and abbreviated versions for action commands like `remove`, `list` etc ...
https://github.com/spf13/cobra/blob/master/command.go#L44-L45

**Comment by MichaelMure** [comment]
The opposite argument could be made by saying that adding confusion with each time two commands doing the same thing require a good reason.

**Comment by strk** [comment]
On Mon, Oct 28, 2019 at 11:53:23AM -0700, Michael Muré wrote:
> The opposite argument could be made by saying that adding confusion with each time two commands doing the same thing require a good reason.

A good reason to keep "ls" is "backward compatibility"
while a good reason to have "list"/"remove" is "consistency with other
git commands" (think `git worktree list`, `git worktree remove`, `git
remote remove`)

**Comment by afranke** [comment]
For reference, it is `git remote remove` and `git notes remove`. I am not aware of a core subcommand that has an explicit list action, neither with `ls` nor with `list`. The subcommands I can think of that have a list action are all done when there is no explicit action (`git branch`, `git remote`…).

**Comment by felsgaertner** [comment]
Hi, from my user's perspective, I can just add a statement for the UX part.

Everytime I ask "git bug list", it responds with "did you mean ls?"... Git has a lot of real words as commands. I would vote for non abbreviated commands.

**Comment by MichaelMure** [comment]
One thing to consider here is that I've been considering disconnecting git-bug from the main git command (that is, it would not be a porcelain command anymore), partially because of the git's author complaint about using the "bug" namespace and partially because the commands now can be quite long (`git bug bridge pull` for example could be changed to, say, `gb bridge pull`).

In this case, the argument of coherency with git is a bit moot.

**Comment by strk** [comment]
> `gb bridge pull`

`gub` would be fun (Git-BUg sharada, and reverse of `bug`)
`bug` would also be friendly :)

**Comment by afranke** [comment]
> In this case, the argument of coherency with git is a bit moot.

Not really. People still expect some consistency across tools, especially if they are used in the same context. Look at Unix command line utilities all using `-v` for verbose (and think of how infuriating it is when someone decides that it stands for `--version`) or other similar behaviours. If `gub` (I like that one) and git both use e.g. remove, that’s one less thing I have to keep reminding me of.

**Comment by felsgaertner** [comment]
> the git's author complaint about using the "bug" namespace 

Is that really a big topic? IMHO "bug" is just an alias, whose name I can adjust to my favor. That could also be "git issue" or anything else...

**Comment by MichaelMure** [comment]
See this thread when I first released git-bug: https://marc.info/?l=git&m=153454720614705&w=2
